R/bm-write-csv.R

```diff
@@ -0,0 +1,74 @@
+#' Benchmark CSV writing
+#'
+#' @section Parameters:
+#' * `source` A CSV file path to write to
+#' * `writer` One of `c("arrow", "data.table", "vroom", "readr",)`
+#' * `input` One of `c("arrow_table", "data_frame")`
+#'
+#' @export
+write_csv <- Benchmark(
+  "write_csv",
+  setup = function(source = names(known_sources),
+                   writer = c("arrow", "data.table", "vroom", "readr", "utils"),
+                   input = c("arrow_table", "data_frame")) {
+    writer <- match.arg(writer)
+    input <- match.arg(input)
+
+    # source defaults are retrieved from the function definition (all available
+    # known_sources) and then read the source in as a data.frame
+    source <- ensure_source(source)
+    df <- read_source(source, as_data_frame = match.arg(input) == "data_frame")
+
+    # Map string param name to functions
+  BenchEnvironment(
```

Comment:
```suggestion
    BenchEnvironment(
```

Super minor style thing